### PR TITLE
Hack Day: Add information about the open video sessions

### PR DESCRIPTION
**Pull request by sejas**
- Related https://github.com/wp-cli/wp-cli/issues/5856

# Description

I think adding a the information about the time when the open video sessions will occur is key.

### CODE REVIEW

File: contributor-day.md [hunk 1]
```diff
@@ -12,6 +12,15 @@ Your leads for the day are: [schlessera](https://github.com/schlessera), [daniel
 
 A few seasoned WP-CLI contributors are also helping out: TBD
 
+## Open Video Sessions
+
+During the Hack Day, we’ll have two open video chat sessions:
+
+- From 🕓 08:00 WET to 09:00 WET
+- From 🕓 15:00 WET to 16:00 WET
```
Comment (swissspidy) [inline contributor-day.md:20]: Let's show this in multiple timezones to make it easier for folks

```suggestion
- From 🕓 00:00-01:00 PST / 03:00-04:00 EST / 08:00-07:00 UTC /  09:00-10:00 CET / 17:00-18:00 JST
- From 🕓 07:00-08:00 PST / 10:00-11:00 EST / 15:00-16:00 UTC / 16:00-17:00 CET / 00:00-01:00 JST
```

I hope these are correct 😄

**Comment by sejas** [comment]
@swissspidy, thanks for the suggestion of adding other timezones.
I deleted the word `From` and added a link to `timeanddate`.

Thank you!

**Review by swissspidy — APPROVED** [review]

File: contributor-day.md [hunk 2]
```diff
@@ -10,6 +10,17 @@ We put together this guide to make contributing as straightforward as possible.
 
 Your leads for the day are: [schlessera](https://github.com/schlessera), [danielbachhuber](https://github.com/danielbachhuber), [swissspidy](https://github.com/swissspidy)
 
+A few seasoned WP-CLI contributors are also helping out: TBD
+
+## Open Video Sessions
+
+During the Hack Day, we’ll have two open video chat sessions:
+
+- [🕓 00:00-01:00 PST / 03:00-04:00 EST / 08:00-07:00 UTC / 09:00-10:00 CET / 17:00-18:00 JST](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20231110T0800)
```
Comment (swissspidy) [inline contributor-day.md:19]:
```suggestion
- [🕓 00:00-01:00 PST / 03:00-04:00 EST / 08:00-09:00 UTC / 09:00-10:00 CET / 17:00-18:00 JST](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20231110T0800)
```

**Comment by sejas** [comment]
@swissspidy , could we merge this one?
- https://github.com/wp-cli/handbook/pull/475

I introduced a deleted line by mistake.

Thanks for reviewing and merging !! 🙏

**Pull request merged**